app/views/courses/moss.html.erb

```diff
@@ -2,184 +2,172 @@
 
 <h2>Run the Moss Cheat Checker</h2>
 
-<% if @course.assessments.empty? %>
-  <ul class="collection" style="text-align: center; font-size: 18px;">
-    <li class="collection-item add">
-      <p>You do not seem to have any assessments in this course to be able to run the Cheat Checker</p>
-      <%= link_to "<i class='material-icons left'>assignment</i> Install Assessment".html_safe, install_assessment_course_assessments_path(@course), { title: "Install Assessment", class: "btn btn-large red darken-3" } %>
-    </li>
-  </ul>
-<% else %>
-  <% content_for :javascripts do %>
-    <script type="text/javascript">
-      /**
-       * toggles the "style.display" of the element with the given id.
-       **/
-      function toggleOptions(dropdownId, tableId) {
-        let show = document.getElementById(tableId);
-        show.style.display = (show.style.display == "none")? "block" : "none";
-        renderDropDownIcon(dropdownId, tableId);
-      }
-    </script>
-  <% end %>
+<% content_for :javascripts do %>
+  <%= javascript_include_tag "moss" %>
+<% end %>
+
+<strong>Filter Courses (Keywords Separated by Space): </strong>
+<input id="courseFilter" placeholder="Course Name" type="text" value="<%= @course.display_name %>" style="margin-bottom: 10px;" autocomplete="off">
 
-  <h4>Step 1:</h4>
-  <p>Check the box for each assessment you want to send to Moss for cheat checking.</p>
+<h4>Step 1:</h4>
+<p>Check the box for each assessment you want to send to Moss for cheat checking.</p>
 
-  <%= form_tag(run_moss_course_path(@course), multipart: true) do %>
-    <ul class="moss-list">
-      <li id="<%= @course.display_name %>">
-        <div class="moss-dropdown" id="moss-dropdown" onclick="toggleOptions('moss-dropdown', '<%= @course.id %>Assessments')">
+<%= form_tag(run_moss_course_path(@course), multipart: true) do %>
+  <ul class="moss-list">
```

Comment:
Right now the dropdowns have no white space between each other, so they're bunched. You could fix this by moving the background color to each li and adding margins between each

Reply:
Added spacing